### PR TITLE
Code cleanup of Switchunits module

### PR DESCRIPTION
**Pull request by MikeMatrix**
Code cleanup of the Switchunits module.

### CODE REVIEW

**Comment by Glowbal** [comment]
Looks ok :+1:

File: addons/switchunits/functions/fnc_switchUnit.sqf [hunk 1]
```diff
@@ -49,21 +44,12 @@ if (_leave) exitWith {
...
-DFUNC(pfhSwitchUnit) = {
-
-    private ["_args", "_unit", "_oldUnit", "_respawnEhId", "_oldOwner"];
-    _args = _this select 0;
-
-    _unit = _args select 0;
-    _oldUnit = _args select 1;
-
-
+_fnc_pfhSwitchUnit = {
```
Comment (jokoho48) [inline addons/switchunits/functions/fnc_switchUnit.sqf]: missing Private?

File: addons/switchunits/functions/fnc_switchBack.sqf [hunk 2]
```diff
@@ -10,20 +10,19 @@
...
 
-PARAMS_1(_originalPlayerUnit);
+params ["_originalPlayerUnit"];
 
 [_originalPlayerUnit] joinSilent GVAR(OriginalGroup);
 
-DFUNC(pfhSwitchBack) = {
-    PARAMS_2(_args,_pfID);
-    EXPLODE_2_PVT(_args,_originalPlayerUnit,_currentUnit);
+_fnc_pfhSwitchBack = {
```
Comment (Glowbal) [inline addons/switchunits/functions/fnc_switchBack.sqf]: Missing private. Might be better to just put the code straight into the PFH without the assignment though

File: addons/switchunits/functions/fnc_markAiOnMap.sqf [hunk 3]
```diff
@@ -10,28 +10,24 @@
...
+params ["_sidesToShow"];
 
 GVAR(AllMarkerNames) = [];
 
-DFUNC(pfhMarkAiOnMap) = {
-    private ["_args", "_sides"];
-    _args = _this select 0;
-    _sides = _args select 0;
-
+_fnc_pfhMarkAiOnMap = {
```
Comment (Glowbal) [inline addons/switchunits/functions/fnc_markAiOnMap.sqf]: Also a missing private. Same thing, as with the other, might be better to put the code straight into the PFH without an assignment.

**Pull request merged**